app/commodities/page.tsx

```diff
@@ -1,12 +1,31 @@
 import { Metadata } from 'next';
 import PageClient from './page.client';
+import { ICommodity } from '../_types';
 
 export const metadata: Metadata = {
   title: 'EDPN - Commodities',
   description: 'Elite Dangerous Pilots Network',
   icons: 'EDPN_logo_dark_background.png',
 };
 
-export default function Page() {
-  return <PageClient />;
+export default async function getCommodities() {
+  let commodities: ICommodity[] | null = null;
+
+  try {
+    const commoditiesReq = await fetch(
+      `${process.env.NEXT_PUBLIC_STAGING_API_URL}/api/v1/trade/commodity`,
+      { next: { revalidate: 86400 } },
+    );
+
+    if (!commoditiesReq.ok) {
+      throw new Error(`HTTP error! status: ${commoditiesReq.status}`);
```

Comment:
This error is unhandled and fails the page load if fetch doesn't work. You can return an error response to the UI or for now just return empty to avoid breaking the UI. Check playground on how I have done this for the two server side calls. I am working on defining the pattern to handle both server side and client side (SWR based) fetches, that should introduce some standard.